pom.xml

```
@@ -58,6 +58,11 @@
             <artifactId>mapstruct</artifactId>
             <version>${mapstruct.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.liquibase</groupId>
+            <artifactId>liquibase-core</artifactId>
+            <version>4.27.0</version>
```

```suggestion
            <version>${liquibase.version}</version>
```

spring-boot-parent provides this version